apps/server/src/routers/healthRouter.ts

```diff
@@ -27,13 +27,15 @@ router.get('/', (_req, res) => {
 			appStatus: 'Up',
 			dbStatus: dbHealth.status,
 		};
-		return res.status(200).send(resBody);
+		res.status(200).send(resBody);
+		return;
```

Comment:
naive question: return is implicit, what's the purpose of putting it separately from the response? surely `return res...` has the exact same effect without looking unnecessarily verbose.

Reply:
ugh Express! le why you le break? 😞 
it was a nice pattern! are you sure it's not just their TS implementation that's the problem?